omnibus_build.sh

```diff
@@ -9,11 +9,22 @@
 PROJECT_DIR=dd-agent-omnibus
 PROJECT_NAME=datadog-agent
 LOG_LEVEL=${LOG_LEVEL:-"info"}
+JMX_VERSION=${JMX_BRANCH:-"current"}
+export AGENT_BRANCH=${AGENT_BRANCH:-"master"}
 export OMNIBUS_BRANCH=${OMNIBUS_BRANCH:-"master"}
 export OMNIBUS_SOFTWARE_BRANCH=${OMNIBUS_SOFTWARE_BRANCH:-"master"}
 export OMNIBUS_RUBY_BRANCH=${OMNIBUS_RUBY_BRANCH:-"datadog-5.5.0"}
 export INTEGRATION_CORE_BRANCH=${INTEGRATION_CORE_BRANCH:-"master"}
 
+REMOTE_AGENT_REPO_RAW="https://raw.githubusercontent.com/DataDog/dd-agent"
+LOCAL_DD_AGENT="/dd-agent-repo"
+
+unset_jmx() {
+    echo "JMX provided by dd-agent in selected branch, JMX environment variables will be ignored."
```

Comment:
So, since indeed the `JMX_VERSION` will just be ignored with other pinned versions, we don't need to unset anything. We can cleanup the script.